src/integer/z/from.rs

```diff
@@ -609,6 +649,32 @@ mod tests_from_modulus {
 
 #[cfg(test)]
 mod test_from_fmpz {
+    use flint_sys::fmpz::{fmpz, fmpz_set_ui};
+
+    use super::Z;
+
+    /// Ensure that `from_fmpz` is available for small and large numbers
+    #[test]
+    fn small_numbers() {
+        let fmpz_1 = fmpz(0);
+        let fmpz_2 = fmpz(100);
+
+        assert_eq!(Z::from_fmpz(fmpz_1), Z::ZERO);
+        assert_eq!(Z::from_fmpz(fmpz_2), Z::from(100));
+    }
+
+    /// Ensure that `from_fmpz` is available for small and large numbers
+    #[test]
+    fn large_number() {
+        let mut fmpz_1 = fmpz(0);
+        unsafe { fmpz_set_ui(&mut fmpz_1, u64::MAX) }
+
+        assert_eq!(Z::from_fmpz(fmpz_1), Z::from(u64::MAX));
+    }
```

Comment:
use same structure as in from_fmpz or use there the same structure as here

Reply:
```suggestion
    #[test]
    fn small_numbers() {
        let fmpz_1 = fmpz(0);
        let fmpz_2 = fmpz(100);

        assert_eq!(Z::from_fmpz(fmpz_1), Z::ZERO);
        assert_eq!(Z::from_fmpz(fmpz_2), Z::from(100));
    }

    /// Ensure that `from_fmpz` is available for small and large numbers
    #[test]
    fn large_numbers() {
        let mut fmpz_1 = fmpz(0);
        unsafe { fmpz_set_ui(&mut fmpz_1, u64::MAX) }

        assert_eq!(Z::from_fmpz(fmpz_1), Z::from(u64::MAX));
    }
```